### PR TITLE
Add Software-Defined Networking

### PR DESCRIPTION
**Pull request by TakeshiTseng**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for the suggestion. Would you mind reading the [contributing guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [awesome manifest](https://github.com/sindresorhus/awesome/blob/master/awesome.md) :)

In the description `A awesome list about Software Defined Network (SDN)` you need to linkify SDN to a page that explains what it is. I have no idea what it is and as a user would probably leave right away. It would also help to include a succinct intro to what SDN is.

**Comment by TakeshiTseng** [comment]
Hello,
Thanks for your advice. I add introduction section to the list, please check.

File: readme.md [hunk 1]
```diff
@@ -195,6 +195,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Open Source Society University](https://github.com/open-source-society/computer-science)
 - [Functional Programming](https://github.com/lucasviola/awesome-functional-programming)
 - [Static Analysis & Code Quality](https://github.com/mre/awesome-static-analysis)
+- [Software Defined Network](https://github.com/sdnds-tw/awesome-sdn)
```
Comment (sindresorhus) [inline readme.md]: According to Wikipedia it's `Software-Defined Networking`

You're also very inconsistent on the usage in your readme. Try to make it more consistent.

**Comment by sindresorhus** [comment]
Added.

**Pull request closed without merging**